### ISSUE TITLE
Make it a taskbar app (as originally thought)

### ISSUE THREAD

**Issue by Tbaut** [paritytech/fether#312]
Talking with @bjornwgnr about it.
We ditched the idea to make Fether a task bar/tray app at some point because it wasn't looking good on linux and windows (can't find the issue). I think we should spend some time on it again. It would potentially allow a much higher adoption because it can be launched on startup and sync comfortably while always be at hand (synced) to be used when needed... The inherent problems of using a light client are invisible because regularly synced in background.

**Comment by amaurymartiny** [comment]
The relevant library: https://github.com/maxogden/menubar

Unforunately it doesn't seem so active anymore

**Comment by ltfschoen** [comment]
@amaurymartiny @Tbaut I had a go at this in branch https://github.com/paritytech/fether/compare/luke-312-taskbar-app?expand=1

<img width="401" alt="screen shot 2019-01-04 at 1 46 46 pm" src="https://user-images.githubusercontent.com/6226175/50671679-7ee1b180-1027-11e9-90ce-bbc640e68d73.png">

If I click the icon to show the taskbar app, and then go to View > Toggle Developer Tools, it displays:
<img width="688" alt="screen shot 2019-01-04 at 1 50 34 pm" src="https://user-images.githubusercontent.com/6226175/50671723-b18baa00-1027-11e9-9808-aaedd2f46c72.png">

I think this is because previously in the fether-electron package's package.json the `electron` binary was called with `"electron": "cross-env SKIP_PREFLIGHT_CHECK=true electron dist/main/main.js",`, but now i it's called with `"electron": "cross-env SKIP_PREFLIGHT_CHECK=true electron mb.js",` and mb.js doesn't start a parity-ethereum node to expose that WebSocket port.

So I started a parity-ethereum node with the following, but it doesn't trust the origin from `file://`:
```
scon @ ~/code/src/paritytech/parity-ethereum - [stable] $ ./target/release/parity --chain kovan --light
2019-01-04 13:47:59  Starting Parity-Ethereum/v2.1.9-stable-af1169d2f-20181205/x86_64-macos/rustc1.30.1
2019-01-04 13:47:59  Keys path /Users/scon/Library/Application Support/io.parity.ethereum/keys/kovan
2019-01-04 13:47:59  DB path /Users/scon/Library/Application Support/io.parity.ethereum/chains_light/kovan/db/9bf388941c25ea98
2019-01-04 13:47:59  Running in experimental Light Client mode.
2019-01-04 13:48:00  Debug API is not available in light client mode.
2019-01-04 13:48:00  Debug API is not available in light client mode.
2019-01-04 13:48:04  Public node URL: enode://25d97eab21b251a819a21cf914ded0493786fa4bc12c5906c455b2a7060d4d00a03c669286ce504438bdecb39942f2dc6e431231d0b53fc9655cc776b12f69cd@172.20.10.4:30303
2019-01-04 13:48:05  Syncing #9938926 0xbcff…74c9     0.0 hdr/s      0+    0 Qed  12/50 peers   664 bytes cache 0 bytes queue  RPC:  0 conn,    0 req/s,    0 µs
2019-01-04 13:48:08  Imported #9941183 0x8082…b4ff (0.47 Mgas) + another 154 header(s)
2019-01-04 13:48:08  Blocked connection to WebSockets server from untrusted origin: Some("file://")
```

**Comment by ltfschoen** [comment]
Oh I just had to pass the option to trust that origin:
```
./target/release/parity --chain kovan --light --ws-origins=file://*
```

**Comment by amaurymartiny** [comment]
Closed by #332
